lib/Backend/amd64/RegList.h

```diff
@@ -127,6 +131,11 @@ REG_XMM_ARG(4, XMM4)
 REG_XMM_ARG(5, XMM5)
 REG_XMM_ARG(6, XMM6)
 REG_XMM_ARG(7, XMM7)
+
+#ifndef INT_ARG_REG_COUNT
+#define INT_ARG_REG_COUNT 4
+#endif
+
 #endif  // !_WIN32
```

Comment:
Don't think you need this twice. ;)

Reply:
The reason I have it twice here is that the number _really_ should be 6 on System V, since the calling convention there is to have 6 ints/pointers passed in regsiters. I wasn't sure about how this would interact with the machinery that is currently used on xplat for their custom calling convention machinery, so I changed it back to 4 to keep the current behavior. @Jianchun, any thoughts?